.spelling

```diff
@@ -98,6 +98,7 @@ Kops
 Kuat
 Kube
 Kubecon
+kubectl
```

Comment:
I had to re-add this because the style-guide uses "kubectl run" in an example of what _not_ to do re: formatting.